src/services/callApi.js

```diff
@@ -0,0 +1,24 @@
+import {
```

Comment:
Good practice, just make sure to handle `Cannot destructure vlaue "<value>" of "undefined"` when your request responds with something unexpected.